ql/src/test/results/clientpositive/perf/tez/constraints/cbo_query84.q.out

```diff
@@ -56,24 +56,24 @@ CBO PLAN:
 HiveProject(customer_id=[$0], customername=[$1])
   HiveSortLimit(sort0=[$2], dir0=[ASC], fetch=[100])
     HiveProject(customer_id=[$2], customername=[$6], c_customer_id=[$2])
-      HiveJoin(condition=[=($8, $4)], joinType=[inner], algorithm=[none], cost=[not available])
-        HiveJoin(condition=[=($1, $3)], joinType=[inner], algorithm=[none], cost=[not available])
-          HiveJoin(condition=[=($0, $1)], joinType=[inner], algorithm=[none], cost=[not available])
+      HiveJoin(condition=[=($8, $4)], joinType=[inner], algorithm=[CommonJoin], cost=[not available])
```

Comment:
This is not nice, it seem a side effect of using the new cost model. I am wondering why we did not hit it before? Are we changing the metadata provider correctly?

Reply:
Added logic to reset the `joinAlgorithm` back to `none` in all `HiveJoin` nodes after calculating the cost and switching back the MetadataProvider